specs/electra/validator.md

```diff
@@ -220,4 +221,30 @@ with updated field assignments:
 
 - Set `attestation_data.index = 0`.
 - Let `aggregation_bits` be a `Bitlist[MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT]` of length `len(committee)`, where each bit set from each individual attestation is set to `0b1`.
-- Set `attestation.committee_bits = committee_bits`, where `committee_bits` has the bit set corresponding to `committee_index` in each individual attestation.
+- Set `attestation.committee_index = committee_index`, where `committee_index` is the `committee_index` in each individual attestation.
```

Comment:
Making it the same as in "Construct attestation" would be easier. Reordering this to be the second one as in "Construct attestation" would even be better.
```suggestion
- Set `attestation.committee_index` to the index associated with the validator's committee.
```